### ISSUE TITLE
False positive no-member for talib library

### ISSUE THREAD

**Issue by sylv3r** [PyCQA/pylint#3005]
### Steps to reproduce

- Install talib library (having TA-Lib already installed on your system)

  `pip install TA-Lib`

- Try talib basic example (https://github.com/mrjbq7/ta-lib#function-api)

```python
import numpy
import talib

close = numpy.random.random(100)
output = talib.SMA(close)

print(output)
```

### Current behavior

The line `output = talib.SMA(close)` raise a
`E1101: Module 'talib' has no 'SMA' member (no-member)`
while the code works perfectly fine.

### Expected behavior

Since the code works and that's how the library promotes to use it, this error should not appear.

### pylint --version output

```
pylint 2.3.1
astroid 2.2.5
Python 3.7.4 (default, Jul  9 2019, 23:23:07) 
[Clang 10.0.1 (clang-1001.0.46.4)]
```

**Comment by PCManticore** [comment]
`talib` seems to be a C extension, in which case pylint does not have a clue how to figure out that the said object actually exist. You can use `--extension-pkg-whitelist=talib` to tell it to load the library to build AST from the live module. Or you can ignore the errors with `--ignored-modules=talib`.
